src/components/toolbox/select/index.js

```diff
@@ -11,13 +11,23 @@ class Select extends React.Component {
 
     this.state = {
       isOpen: false,
-      selected: props.selected,
+      selected: '',
     };
 
     this.setSelected = this.setSelected.bind(this);
     this.toggleIsOpen = this.toggleIsOpen.bind(this);
   }
 
+  componentDidMount() {
+    this.setState({ selected: this.props.selected });
+  }
+
+  componentDidUpdate(prevProps) {
+    if (prevProps.selected !== this.props.selected && this.props.selected === '') {
+      this.setState({ selected: this.props.selected });
+    }
+  }
+
```

Comment:
💅 I think `this.props.selected` can be used directly instead of mirorring it to `this.state.selected`

Reply:
I removed the internal `selected` state and refactored the component so that it can work with the new styles in the rest of the instances.